docs/guide/overview.asciidoc

```diff
@@ -38,6 +36,42 @@ This data is buffered for a short period and sent on to APM Server.
\ No newline at end of file
+*Elasticsearch* is a highly scalable open-source full-text search and analytics engine.
+It allows you to store, search, and analyze big volumes of data quickly and in near real time.
+
+*Kibana* is an open source analytics and visualization platform designed to work with Elasticsearch.
+You use Kibana to search, view, and interact with data stored in Elasticsearch indices.
+You an use Kibana to visualize APM data by utilizing the dedicated APM UI bundled in Basic license,
+or the pre-built, open source,
+Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
+
+[[concepts]]
+[float]
+=== APM Concepts
+
+*Agents*
+APM Agents capture different types of information from within their instrumented applications, known as events. Events can be <<transactions,`Transactions`>>, <<transaction-spans,`Spans`>>, and <<errors,`Errors`>>.
+These events are then streamed to a single endpoint in the APM Server which sorts and processes the events. 
+
+* <<transactions,`Transactions`>> - A transaction describes an event captured by an Elastic APM agent instrumenting a service. For example: an HTTP request, or a background job.
+* <<transaction-spans,`Spans`>> - Transactions can have 0, 1, or many spans. A span contains information about a specific code path that has been executed as part of a transaction.
+* <<errors,`Errors`>> - An error event contains information about the original exception that occurred or about a log created when the exception occurred.
+
+*Server*
+The APM Server is a {apm-server-ref}/why-separate-component.html[separate component by design] - it helps keep the agents light, prevents certain security risks, and improves compatibility across the Elastic and APM Stack.  
+
+The {apm-server-ref}/intake-api.html[Intake API] is what we call the internal protocol that APM agents use to talk to the APM Server. 
+After the APM Server has sorted and processed events from the APM agents (via the intake API), the server transforms the data into Elasticsearch documents and stores them in corresponding {apm-server-ref}/exploring-es-data.html[Elasticsearch indices]. In a matter of minutes you can start viewing your application performance data in the {kibana-ref}/apm-getting-started.html[Kibana UI] using our prebuilt dashboards.
+
+*Real User Monitoring (RUM)*
+Real User Monitoring captures user interaction with clients such as web browsers. The {apm-rum-ref}[JavaScript Agent] is Elastic’s RUM Agent. To use it you need to {apm-server-ref}/rum.html[enable RUM support] in the APM Server.
+
+Unlike Elastic APM backend agents which monitor requests and responses, the RUM JS agent monitors the real user experience and interaction within your client-side application. The RUM JS agent is also framework-agnostic, which means it can be used with any frontend JavaScript application.
```

Comment:
Do we want to abbreviate JavaScript as JS in a product name like this?

Reply:
Fair. I had it like that because that's how it's referenced almost everywhere else, but I think you're right to expand in this location